packages/ui/src/memberships/components/MemberProfile/MemberBounds.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import { useBalance } from '@/accounts/hooks/useBalance'
+import { TokenValue } from '@/common/components/typography'
+import { isDefined } from '@/common/utils'
+
+import { UnknownAccountInfo } from '../../../accounts/components/UnknownAccountInfo'
+
+import { AccountMemberRow } from './MemberAccounts'
+
+type Props = {
+  account: string
+}
+
+export const MemberBounds = ({ account }: Props) => {
+  const balance = useBalance(account)
+  return (
+    <div>
+      {!!account && (
+        <AccountMemberRow>
+          <UnknownAccountInfo address={account} placeholderName="Root Account" />
+          <TokenValue value={balance?.total} isLoading={!isDefined(balance?.total)} />
+        </AccountMemberRow>
```

Comment:
This logic id duplicated 3 times: 1 one for root, 1 for controller and once in this component. I think we should adjust this component to be more generic and also use it for root and controller accounts